src/sbvr-api/permissions.ts

```diff
@@ -518,6 +518,11 @@ const generateConstrainedAbstractSql = (
 		odata,
 	);
 
+	if (collapsePermissionFilters == null) {
+		// full access
```

Comment:
I'd like to expand on this as
```suggestion
		// If we have full access then there's no need to provide a constrained definition, just use the table directly
```
since it'll help a lot with understanding why full access == return [undefined]